packages/core/src/app.ts

```diff
@@ -119,11 +119,44 @@ export class Index extends Serializable {
         return this.items.filter((item) => item.hosts.some((h) => h === hashedHost)).length;
     }
 
+    splitHostBySubdomains(host: string) {
```

Comment:
Let's rename this function to something like `getHostnameVariants` and let it return **all** variants, including the actual host name, and the variants with and without `www`. I think that makes it cleaner and easier to understand what's going on.